static/js/reactions.js

```diff
@@ -270,13 +273,26 @@ export function add_reaction(event) {
         emoji_name: event.emoji_name,
         emoji_code: event.emoji_code,
         user_id,
+        vote_text: undefined,
     };
 
     if (r) {
         opts.user_list = r.user_ids;
+        /*
+            setting vote_text here is not really required since it is
+            updated when we call `update_vote_text_on_message` later.
+            This is just to make sure we see the correct value inside
+            the node tests.
+        */
```

Comment:
Can you be precise about which `node` test is checking this detail? I wonder if it'd be better to tweak that.

Reply:
> Can you be precise about which `node` test is checking this detail?

Its being checked as part of the `opts` object in the `"add_reaction/remove_reaction"` test.

> I wonder if it'd be better to tweak the tests to not do this -- generally the goal should be to write clear, correct, code, and then figure out how to test it.

Hmm... the `vote_text` parameter is a part of the `opts` object so if we decide to not set it, its value will just be `undefined` inside the test. I'll study the code and get back to you on this.